frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaParticipants.yml

```diff
@@ -0,0 +1,28 @@
+name: Technical competence criteria
+question: Technical competence criteria
+question_advice: |
+  Technical competence is how well the supplier can do the work you need them to do.
+
+  Evaluate suppliers’ technical competence using criteria for:
+
+  - essential skills and experience
+  - nice-to-have skills and experience
+  - proposal
+
+  **Scoring (optional)**<br />
+  If you want to score individual criteria, you must add points for everything. Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
```

Comment:
You're right. This should be exactly the same message with tweaked eg. And the same heading too.

'Giving points for individual criteria (optional)

You can give points to individual criteria, eg ‘have experience recruiting participants with low digital literacy – 15 points’. If you choose to add points, you must do it for all individual criteria.

The points you give for essential skills and experience and proposal criteria must be higher than the points for nice-to-have skills and experience criteria. Read how to score individual criteria.'